src/cli/lint/annotate/github/tsc.ts

```diff
@@ -1,14 +1,20 @@
 import * as GitHub from '../../../../api/github';
 import { StreamInterceptor } from '../../../../cli/lint/external';
 
+const ansiPattern = [
```

Comment:
Copy pasted from `strip-ansi` and `ansi-regex` packages. Happy to import them instead if we want, however they are only esmodules and I couldn't quite get it working @72636c 

https://github.com/chalk/ansi-regex/blob/02fa893d619d3da85411acc8fd4e2eea0e95a9d9/index.js#L1

Reply:
I think we can use an older major version here. Only the latest should be ESM-only.

Migrating skuba itself to ESM is something we can tackle later and is tracked in #398.